tests/baselines/reference/exportDefaultEnumTargetES2015ModuleAMD.js

```diff
@@ -0,0 +1,128 @@
+//// [tests/cases/compiler/exportDefaultEnumTargetES2015ModuleAMD.ts] ////
+
+//// [a.ts]
+// https://github.com/Microsoft/TypeScript/issues/3792
+
+/**
+ * export default enum A 1 comment
+ */
+export default enum A { FOO } // export default enum A 1 trailing comment
+
+/**
+ * export default enum A 2 comment
+ */
+export default enum A {
+    BAR = 2
+} // export default enum A 2 trailing comment
+
+const enumAUsage = A.FOO;
+
+/**
+ * export enum B 1 comment
+ */
+export enum B {
+    X = 0,
+} // export enum B 1 trailing comment
+
+// export enum B 2 comment
+export enum B {
+    Y = 1
+} // export enum B 2 trailing comment
+
+const enumBUsage = B.Y
+
+namespace ns {
+    // enum ns.C 1 comment
+    export enum C {
+        X = 0
+    } // enum ns.C 1 trailing comment
+    // enum ns.C 2 comment
+    export enum C {
+        Y = 1
+    } // enum ns.C 2 trailing comment
+}
+namespace ns {
+    // enum ns.C 3 comment
+    export enum C {
+        Z = 2
+    } // enum ns.C 3 trailing comment
+
+    const enumCUSage1 = C.X;
+}
+
+namespace ns {
+    const enumCUsage2 = C.X
+}
+
+//// [b.ts]
+import A, { B } from "./a";
+
+const enumAExternalUsage1 = A.FOO;
+const enumAExternalUsage2 = A[enumAExternalUsage1];
+const enumBExternalUsage1 = B.X;
+
+
+//// [a.js]
+// https://github.com/Microsoft/TypeScript/issues/3792
+define(["require", "exports"], function (require, exports) {
+    "use strict";
+    Object.defineProperty(exports, "__esModule", { value: true });
+    /**
+     * export default enum A 1 comment
+     */
+    const A = {};
+    exports.default = A;
+    (function (A) {
+        A[A["FOO"] = 0] = "FOO";
+    })(A); // export default enum A 1 trailing comment
+    /**
+     * export default enum A 2 comment
+     */
+    (function (A) {
+        A[A["BAR"] = 2] = "BAR";
+    })(A); // export default enum A 2 trailing comment
+    const enumAUsage = A.FOO;
+    /**
+     * export enum B 1 comment
+     */
+    const B = {};

```

Comment:
`B` is never exported, despite the fact it is declared as `export enum B` in the test.